cypress/integration/materialReceipt/create_material_receipt_with_quality_issue.js

```diff
@@ -136,7 +140,7 @@ describe('Create test: create material receipt with quality issue, https://githu
       .should('have.value', '0.1')
       .clear()
       .type('10{enter}');
-    cy.get('#lookup_M_Product_ID .input-dropdown').should('not.have.class', 'input-block');
+    cy.waitUntilProcessIsFinished();
     /**Complete purchase order */
```

Comment:
I have added some waits back as my previous solution did not work anymore; but as I understood @kuba is investigating some websocket alternative to these waits - I will also try to find a solution after I finish my current tests I am working on. Unfortunately at the moment I don't have a replacement to those waits:(